appointment-management/src/main/java/org/dljl/dto/CreateBlockDTO.java

```diff
@@ -0,0 +1,38 @@
+package org.dljl.dto;
+
+import java.time.LocalTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateBlockDTO {
+  private Long providerId;
+  private LocalTime startTime;
+  private LocalTime endTime;
+
+  // Getters and setters
+  public Long getProviderId() {
```

Comment:
With @Getter and @Setter, the Lombok provide default getter and setter, not need to write your own if you don't have special requirements.